### PR TITLE
Add separate package with debug symbols.

### PR DESCRIPTION
**Pull request by johanfforsberg**
Checklist
* [x ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x ] Ensured the license file is being packaged.

These changes add a separate "dbg" package with debug symbols.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -36,11 +40,26 @@ requirements:
...
+        - test -f ${PREFIX}/lib/libhdb++timescale${SHLIB_EXT}
+        - test -f ${PREFIX}/lib/cmake/libhdbpp-timescale/libhdbpp-timescaleConfig.cmake
 
+      
+  - name: "libhdbpp-timescale-dbg"
+    requirements:
+      run:
+        - "{{ pin_subpackage('libhdbpp-timescale', exact=True) }}"
+    files:
+      commands:
```
Comment (beenje) [inline recipe/meta.yaml]: `commands` shouldn't be there.

It needs to be under `test`.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -36,11 +40,26 @@ requirements:
...
-test:
-  commands:
-    - test -f ${PREFIX}/lib/libhdb++timescale${SHLIB_EXT}
-    - test -f ${PREFIX}/lib/cmake/libhdbpp-timescale/libhdbpp-timescaleConfig.cmake
+outputs:
+  - name: "libhdbpp-timescale"
+    files:
+      - "lib/libhdb++timescale.so"
+      - "lib/libhdb++timescale.so.2"
+      - "lib/libhdb++timescale.so.{{ version }}"
```
Comment (beenje) [inline recipe/meta.yaml]: All filed have to be listed. We have to specify the cmake files. Specifying the dir should be fine:

lib/cmake/libhdbpp-timescale/

File: recipe/meta.yaml [hunk 3]
```diff
@@ -36,11 +40,26 @@ requirements:
...
+    test:
+      commands:
+        - test -f ${PREFIX}/lib/libhdb++timescale${SHLIB_EXT}
+        - test -f ${PREFIX}/lib/cmake/libhdbpp-timescale/libhdbpp-timescaleConfig.cmake
```
Comment (beenje) [inline recipe/meta.yaml]: This didn't run: `Nothing to test for: /home/conda/feedstock_root/build_artifacts/linux-64/libhdbpp-timescale-2.0.1-h924138e_2.conda`

I think you have to leave them before outputs for this package.

File: recipe/meta.yaml (hunk 2 above)
Comment (johanfforsberg) [reply recipe/meta.yaml]: Not sure what you mean here; should the section be
```yaml
   files:
     - lib/cmake/libhdbpp-timescale/
```

Comment (beenje) [reply recipe/meta.yaml]: Just add that path to the existing section:

```suggestion
      - "lib/libhdb++timescale.so.{{ version }}"
      - lib/cmake/libhdbpp-timescale/
```

**Comment by johanfforsberg** [comment]
@conda-forge-admin, please rerender

File: recipe/build.sh [hunk 4]
```diff
@@ -4,3 +4,9 @@ cmake ${CMAKE_ARGS} \
 
 cmake --build build -j $CPU_COUNT
 cmake --build build --target install
+
```
Comment (beenje) [inline recipe/build.sh:7]: I missed that you should change the `CMAKE_BUILD_TYPE` to `RelWithDebInfo`.

**Pull request merged**